src/MLJIteration.jl

```diff
@@ -39,6 +40,7 @@ include("constructors.jl")
 include("traits.jl")
 include("ic_model.jl")
 include("core.jl")
+include("serialization.jl")
```

Comment:
This `include` logically needs to precede line 36, or else just dump the code in "controls.jl", which already contains the other controls added by this package. I'd prefer the latter, to be honest.